cmd/kyma/alpha/create/module/module.go

```diff
@@ -468,9 +469,9 @@ func (cmd *command) getModuleTemplateAnnotations(modCnf *Config, crValidator val
 
 	isClusterScoped := isCrdClusterScoped(crValidator.GetCrd())
 	if isClusterScoped {
-		annotations[v1beta2.IsClusterScopedAnnotation] = v1beta2.EnableLabelValue
+		annotations[shared.IsClusterScopedAnnotation] = v1beta2.EnableLabelValue
 	} else {
-		annotations[v1beta2.IsClusterScopedAnnotation] = v1beta2.DisableLabelValue
+		annotations[shared.IsClusterScopedAnnotation] = v1beta2.DisableLabelValue
 	}
 	annotations["operator.kyma-project.io/module-version"] = moduleVersion
```

Comment:
can you also replace with reference to shared package? please also check if there is other hardcoded values need to be referenced, feel free create followup issue if some scope becomes too big.